README.md

```diff
@@ -4,6 +4,8 @@ Appnexus Android SDK
 See our documentation on our wiki at 
 https://wiki.appnexus.com/display/sdk/Home
 
+Get the latest release notes here: https://wiki.xandr.com/display/sdk/Android+SDK+Release+Notes
```

Comment:
Docs are now in https://docs.xandr.com/bundle/mobile-sdk/page/android-sdk-release-notes.html